src/main/java/com/socrata/datasync/ui/PortJobTab.java

```diff
@@ -146,6 +152,15 @@ public PortJobTab(PortJob job, JFrame containingFrame) {
         destinationSetIDTextFieldContainer.add(openSinkDatasetButton);
         jobPanel.add(destinationSetIDTextFieldContainer);
 
+        if(useUseNewBackend) {
+            jobPanel.add(UIUtility.generateLabelWithHelpBubble("Use new checkbox", NBE_TIP_TEXT, HELP_ICON_TOP_PADDING));
```

Comment:
I'm having trouble picturing where this text shows up in a way that makes sense.

Reply:
In the main port job panel, right below the "destination dataset id" field